.vscode/settings.json

```diff
@@ -17,4 +17,15 @@
     "**/lib": true,
     "common/temp": true
   },
+  "eslint.workingDirectories": [
+    {
+      "mode": "auto"
+    }
+  ],
+  // "eslint.format.enable": true,
```

Comment:
why is this commended out?

Reply:
bad copypasta. It's not necessary, I removed it.